Resources/Prototypes/_Sunrise/Entities/Objects/Weapons/Guns/Ammunition/Cartridges/heavy_rifle_nt.yml

```
@@ -83,17 +83,94 @@
+    magState: mag
+    steps: 4
+    zeroVisible: false
+  - type: Appearance
+
+- type: entity
+  parent: BaseMagazineBoxHeavyRifleNT
+  id: MagazineBoxHeavyRifleNT
+  name: ammunition box (7,62mm NT)
+  description: Cardboard box full of 7.62x51mm full-power rifle rounds. Dated, but still widely used by marksman rifles and medium machine guns.
+  components:
+  - type: BallisticAmmoProvider
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)